test/verify/check-realms

```diff
@@ -671,7 +671,7 @@ class TestPackageInstall(packagelib.PackageCase):
         b.click("#system-info-domain a")
         b.wait_present("#cockpit_modal_dialog .modal-footer button.btn-primary:not([disabled])")
         # restore realmd service, to pretend that package install completed
-        m.execute("mv /usr/share/dbus-1/system-services/org.freedesktop.realmd.service{.disabled,}")
+        m.execute("mv /usr/share/dbus-1/system-services/org.freedesktop.realmd.service{.disabled,} && systemctl start realmd")
```

Comment:
Erk yes, what a nasty hack. How about doing this more properly? `systemctl stop realmd && systemctl mask realmd` for disabling, and `systemctl unmask realmd` for re-enabling?

Reply:
Sorry, I didn't mean "your" hack, but the one that's already in the test. :sweat_smile:

Reply:
Hmm, who could introduce such nasty hack? :thinking: 
https://github.com/cockpit-project/cockpit/commit/b20ef2d7e337fb4d87f8cb813cc01fb5a04727d3
😜😜

Reply:
But it could work with masking of the service, will check it out.

Reply:
Exactly, that's the only person whose nasty hack I'd call out as such :joy: 

Thanks!